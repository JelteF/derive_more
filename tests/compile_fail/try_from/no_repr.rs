#[derive(derive_more::TryFrom)]
enum Enum {
    Variant
}

fn main() {}
