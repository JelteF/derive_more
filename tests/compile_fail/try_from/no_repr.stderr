error: `#[try_from(repr)]` attribute is expected here
 --> tests/compile_fail/try_from/no_repr.rs:2:6
  |
2 | enum Enum {
  |      ^^^^
